src/compiler/index.js

```diff
@@ -437,6 +450,30 @@ class Compiler {
                 )
             }
 
+            if (protocol.name == 'substreams') {
```

Comment:
This resolves `.spkg` and copy it to `build` directory, when on `Substreams`

Reply:
@dotansimha

Reply:
thanks!